tools/perf/ucc_pt_config.cc

```diff
@@ -1,16 +1,19 @@
 #include "ucc_pt_config.h"
 
 ucc_pt_config::ucc_pt_config() {
-    bootstrap.bootstrap = UCC_PT_BOOTSTRAP_MPI;
-    bench.coll_type     = UCC_COLL_TYPE_ALLREDUCE;
-    bench.min_count     = 128;
-    bench.max_count     = 128;
-    bench.dt            = UCC_DT_FLOAT32;
-    bench.mt            = UCC_MEMORY_TYPE_HOST;
-    bench.op            = UCC_OP_SUM;
-    bench.inplace       = false;
-    bench.n_iter        = 10;
-    bench.n_warmup      = 10;
+    bootstrap.bootstrap  = UCC_PT_BOOTSTRAP_MPI;
+    bench.coll_type      = UCC_COLL_TYPE_ALLREDUCE;
+    bench.min_count      = 128;
+    bench.max_count      = 128;
+    bench.dt             = UCC_DT_FLOAT32;
+    bench.mt             = UCC_MEMORY_TYPE_HOST;
+    bench.op             = UCC_OP_SUM;
+    bench.inplace        = false;
+    bench.n_iter_large   = 1000;
```

Comment:
large should be 20/200 and small 100/1000 - vice versa

Reply:
oops 😀, fixed